internal/pkg/fleetapi/action.go

```diff
@@ -434,7 +436,7 @@ func (a *ActionCancel) AckEvent() AckEvent {
 
 // ActionDiagnostics is a request to gather and upload a diagnostics bundle.
 type ActionDiagnostics struct {
-	ActionID   string `json:"action_id"`
+	ActionID   string `json:"id"`
 	ActionType string `json:"type"`
 	UploadID   string `json:"-"`
```

Comment:
How are we handling unmarshaled fields like this one in the action store? we don't save any action that has unmarshaled fields?

Reply:
sorry, I did not understand what you mean